plugins/parodos/src/components/workflow/hooks/useWorkflows.ts

```diff
@@ -0,0 +1,49 @@
+import { fetchApiRef, useApi } from '@backstage/core-plugin-api';
+import useAsyncFn, { AsyncFnReturn } from 'react-use/lib/useAsyncFn';
+import { WorkflowStatus } from '../../../models/workflowTaskSchema';
+
+export function useWorkflows(
+  workflowsUrl: string,
+): AsyncFnReturn<(projectId: string) => Promise<WorkflowStatus[]>> {
+  const { fetch } = useApi(fetchApiRef);
+
+  return useAsyncFn(
+    async (projectId: string) => {
+      const payload = { projectId };
+      // eslint-disable-next-line no-console
+      console.log(payload);
+
+      await new Promise(resolve => setTimeout(resolve, 1000));
```

Comment:
do we need this?

Reply:
Yeah, because we don't have real API it's just emulates a request